documents-webapp/src/main/webapp/skin/less/documents.less

```diff
@@ -64,6 +68,13 @@
       display: none;
     }
 
+    .v-data-table__mobile-table-row > :last-child {
+      padding: 0;
+      margin: 0;
```

Comment:
You can Add directly the vuetify class ma-0 pa-0 when it is a mobile

Reply:
The element is not reachable since it is included in the v-data-table component.